errors_113.go

```diff
@@ -0,0 +1,70 @@
+// This file implements the helper interfaces for the errors package introduced in Go 1.13
+// +build go1.13
+
+package errorx
+
+import "errors"
+
+// As implements the helper interface for errors.As.
+// If v is a pointer to a pointer to Error, *v will be set to this error.
+// Otherwise, it traverses the error chain until it reaches a non-*Error error and calls errors.As on that error and the
+// target.
+func (e *Error) As(v interface{}) bool {
+	if target, ok := v.(**Error); ok {
+		*target = e
+		return true
+	}
+
+	cause := e
+	for cause != nil {
```

Comment:
Again, in this enumeration we ignore transparency.

Reply:
From my understanding of transparency, that doesn't play a role here.  [The documentation](https://godoc.org/github.com/joomcode/errorx#ErrorBuilder.Transparent) about it indicates that:

>  Transparent wrap hides the current error type from the type checks and exposes the error type of the cause instead.

So we would skip over transparent errors and go straight to the cause, but if something is an `*Error`, we already know that it's not assignable to `v`, so we're just going to the end of the chain and if it's a non-errorx error, we call `errors.As` on that.

Reply:
But it also works the other way around. If an error is wrapped non-transparently, we shouldn't inspect it.